motion_tracker/utils/crop_vis.py

```diff
@@ -51,3 +52,32 @@ def show_img(img, box_coords=None, window_name="Happy Dance Image",
     cv2.waitKey(msec_to_show_for)
     cv2.destroyWindow(window_name)
 
+def show_img_from_filepath(img_filename, imagenet=True): 
+    """Show an image and it's bounding box from a given filepath
+
+    Args: 
+    ----
+        img_filename: str
+        imagenet: boolean 
+            Holds whether it's an imagenet or alov image. 
+    """
+
+    raw_img_dir = 'data/imagenet/images/' if imagenet else 'work/alov/images/'
+    img_metadata_fp = 'work/imagenet/parsed_bb2.csv' if imagenet else 'work/alov/parsed_bb.csv'
+
+    img_metadata_df = pd.read_csv(img_metadata_fp)
+    if imagenet: 
+        img_metadata = img_metadata_df.query("filename == @img_filename")
+        box_coords = Coords(img_metadata.x0, img_metadata.y0, 
+                            img_metadata.x1, img_metadata.y1)
+    else: 
+        img_metadata = img_metadata_df.query("filename_start == @img_filename")
+        box_coords = Coords(img_metadata.x0_start, img_metadata.y0_start, 
+                            img_metadata.x1_start, img_metadata.y1_start)
+
+    img = cv2.imread(raw_img_dir + img_filename)
+    show_img(img, box_coords.as_array())
+
```

Comment:
when in doubt, I'd prefer to take it out. Otherwise it will keep lying around as extra cruft, and we'll never actually notice that it's not getting used.  

If we ever need it in the future, we can find it in our git history.